kuma/static/js/main.js

```diff
@@ -15,7 +15,7 @@
         var result;
         try {
             localStorage.setItem(uid, uid);
-            result = localStorage.getItem(uid) == uid;
+            result = localStorage.getItem(uid) === uid;
```

Comment:
This test now reports false instead of true so I think `==` was the original intent here.  I guess we need to either re-write it to conform or add an exception for the linter :/